client/src/components/Related/ProductCarousel.jsx

```diff
@@ -66,27 +77,60 @@ class ProductCarousel extends React.Component {
       });
   }
 
+  prev() {
+    this.myRef.current.scrollLeft -= 230;
+    if (this.myRef.current.scrollLeft < 230) {
+      console.log('BEGINNING OF SCROLL');
+    }
+  }
+
+  next() {
+    const { productInfo } = this.state;
+    this.myRef.current.scrollLeft += 230;
+    if ((productInfo.length - 3) * 230 === this.myRef.current.scrollLeft) {
+      console.log('END OF SCROLL');
+    }
+  }
+
   render() {
     const { productInfo } = this.state;
     if (productInfo.length === 0) {
       return 'loading...';
     }
     return (
-      <>
-        {productInfo.map((product, i) => (
-          // eslint-disable-next-line react/no-array-index-key
-          <div className="card" key={i}>{product.name} - {product.category}</div>
-        ))}
-      </>
+      <div className="carousel">
+        <div>RELATED PRODUCTS</div>
+
+        <button className="carousel__button carousel__button--left" type="button" onClick={() => this.prev()}>
+          <img src="./images/arrow-left.png" alt="" />
+        </button>
+
+        <div className="carousel__track-container">
+          <ul className="carousel__track" ref={this.myRef}>
+            {productInfo.map((product, i) => (
+              <li className="carousel__slide" key={i}>
+                <div className="card">
```

Comment:
Consider breaking down each carousel item into an individual file in the future

Reply:
Noted! Thank you for reviewing!